policy-service/src/main/java/uk/gov/gchq/palisade/service/policy/model/PolicyResponse.java

```diff
@@ -149,7 +150,7 @@ public static IUserId create() {
         }
 
         /**
-         * Starter method for the Builder class that uses a PolicyRequest and appends the Rules.
+         * Starter method for the Builder class which uses a PolicyRequest and appends the Rules.
```

Comment:
Can't get the comment to focus on the line I want, but on line 157 suggest changing policy-service to Policy Service